src/pages/Swap/Swap.tsx

```diff
@@ -16,7 +28,19 @@ export const Swap: React.FC<SwapProp> = ({
   checkCoinBottom,
   isInstalled,
   isConnected,
+  changeToken,
+  selectError,
 }) => {
+  const [changeFlag, setChangeFlag] = useState<boolean>(false);
+  const [amountToBeSent, setAmountToBeSent] = useState<string>('0.0');
```

Comment:
I would put undefined to start, so the user only see placeholder

Reply:
fixed this